content/millennium/r4/general-clinical/condition.md

```diff
@@ -53,7 +54,7 @@ _Implementation Notes_
  `patient`         | This or `_id` or `subject`     | [`reference`] | Who the condition is for. Example: `12345`
  `subject`         | This or `_id` or `patient`     | [`reference`] | Who the condition is for. Example: `Patient/12345`
  `clinical-status` | No                             | [`token`]     | The clinical status of the condition. Example: `active`, `inactive`, `resolved`
- `category`        | No                             | [`token`]     | The category of the condition. Categories problem-list-item and encounter-diagnosis are supported as of now. Example: `problem-list-item`, `encounter-diagnosis`
+ `category`        | No                             | [`token`]     | The category of the condition. Categories problem-list-item encounter-diagnosis and health-concern are supported as of now. Example: `problem-list-item`, `encounter-diagnosis`, `health-concern`
```

Comment:
Can you check the description with functional Jira

Reply:
Looks like in functional overview it was missed, its fine to add the supported categories

Reply:
nit: can we add , between the categories

Reply:
Addressed